modules/trusted-firmware-m/Kconfig

```diff
@@ -20,6 +20,17 @@ config TFM_BL2
 	bool
 	default n
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
+	default y
+	default on !UART_1_NRF_UARTE
```

Comment:
Typo.

Should be

default y if !UART_1_NRF_UARTE

or

depends on !UART_1_NRF_UARTE

Reply:
Should be fixed now